examples/swagger/swagger/public/definitions/InterestedParty.yaml

```diff
@@ -0,0 +1,11 @@
+allOf:
+  - $ref: 'InvolvedParty.yaml'
```

Comment:
Ok, I guess I thought that the language was vague enough to be acceptable. What if we change 'party' to 'person'?